Analysis/SkimProducer.py

```diff
@@ -33,21 +36,31 @@
   syst_trees = []
   k=0
   for systFile in all_files:
-    if args.test and k>2 : continue
-    k+=1
-    inFileCentralName = os.path.join(args.inputDir, args.centralFile)
-    if args.test: print(inFileCentralName)
+    if args.test and k>=1 :
+      continue
     inFileShiftedName = os.path.join(args.inputDir, systFile)
-    if args.test: print(inFileShiftedName)
+    if args.test: print('shifted file = ', inFileShiftedName)
+    if args.test: print('index = ', k)
     treeName = systFile.strip('.root')
-    cmd = f"""python3 /afs/cern.ch/work/v/vdamante/hhbbTauTauRes/prod/Framework/Analysis/SkimEvents.py --inFileCentral {inFileCentralName} --inFileShifted {inFileShiftedName} --outDir {args.workingDir} --treeName {treeName}"""
+    cmd = f"""python3 Analysis/SkimEvents.py --inFileCentral {inFileCentralName} --inFileShifted {inFileShiftedName} --outDir {args.workingDir} --treeName {treeName}"""
```

Comment:
put full path for py starting with ANALYSIS_PATH

Reply:
ok